robot/basestation/static/js/elements/navigation/navigation.js

```diff
@@ -1,191 +1,545 @@
+                    if (lat && long && heading) {
+                        logInfo('Found antenna parameters to set')
+                        setupAntennaStats(lat, long, heading)
+                    } else {
+                        logInfo('Antenna parameters were not found')
+                    }
+                })
             })
-          } else {
-            appendToConsole(
-              'One or more antenna parameters is missing, if you would like antenna directions then please input them'
-            )
-            $('#antenna-inputs').show()
-            $('#antenna-unchanging').hide()
-          }
         })
-      } else {
-        appendToConsole(
-          'One or more antenna parameters is missing, if you would like antenna directions then please input them'
-        )
-        $('#antenna-inputs').show()
-        $('#antenna-unchanging').hide()
-      }
-    })
-
-    goal_latitude.get(function (val) {
-      if (val != null) {
-        $('#goal-latitude').text(val)
-        goal_longitude.get(function (val) {
-          if (val != null) {
-            appendToConsole(
-              'GPS goal parameters already set, displaying directions to the goal'
-            )
-            $('#goal-longitude').text(val)
-            $('#goal-inputs').hide()
-            $('#goal-unchanging').show()
-          } else {
-            appendToConsole(
-              'One or more GPS goal parameters is missing, if you would like directions to the goal then please input them'
-            )
-            $('#goal-inputs').show()
-            $('#goal-unchanging').hide()
-          }
+
+        goal_latitude.get(function(lat) {
+            goal_longitude.get(function(long) {
+                if (lat && long) {
+                    has_gps_goal.set(true)
+                    logInfo('Found goal coordinates to set')
+                    setupGoalStats(lat, long)
+                } else {
+                    logInfo('Goal parameters not found')
+                }
+            })
+        })
+
+        function setupAntennaStats(lat, long, heading) {
+            $('#antenna-stats-lat').text(lat.toFixed(6))
+            $('#antenna-stats-long').text(long.toFixed(6))
+            $('#antenna-stats-heading').text(heading + '°')
+        }
+
+        function setupGoalStats(lat, long) {
+            $('#goal-stats-lat').text(lat.toFixed(6))
+            $('#goal-stats-long').text(long.toFixed(6))
+        }
+    }
+
+    $('#antenna-select-lat-format-btn').on('click', function(event) {
+        event.preventDefault()
+        createAntennaLatitudeHandler()
+    })
+    $('#antenna-select-long-format-btn').on('click', function(event) {
+        event.preventDefault()
+        createAntennaLongitudeHandler()
+    })
+    $('#antenna-confirm-btn').on('click', function(event) {
+        event.preventDefault()
+
+        let lat = -1
+        let long = -1
+
+        try {
+            if ($("#antenna-lat-fieldset").attr('format') == 'DD') {
```

Comment:
can we change `DD` to something more descriptive? Ideally we want to be able to read variables/codes years later and not have to scratch our heads at what it could stand for. There are some other instances like DDM, DMS, decdeg, etc. The actual names should be spelled out to make it more readable.

Reply:
Done